app/services/queries/cclow/litigation_query.rb

```diff
@@ -204,6 +117,5 @@ def litigation_resources(sql)
         Litigation.where(id: ids)
```

Comment:
One more thing. Here should be `scope.where`, becuase the default scope in this Query object is `Litigation.published`, so to be on the safe side, we cannot show unpublished ones

Reply:
Good point. I'll fix it.

Reply:
done in a new commit, updated specs as well.

Reply:
Ha, wait! that method was not used anymore? Then it could be deleted ;]